src/Filter/FilterWithOperator.php

```diff
@@ -9,9 +9,9 @@ class FilterWithOperator extends AbstractFilter
     /** @var string */
     private $operator;
 
-    public function __construct(string $column, Value $value, string $operator)
+    public function __construct(string $column, Value $value, string $operator, string $title)
```

Comment:
Any reason for different parameter order than in AbstractFilter?

Reply:
Well actually there is none.. I did not care about it.. should I unify this?